_i18n/en/_posts/2018-01-13-bus.md

```diff
@@ -3,15 +3,15 @@ author: tungfa
 layout: post
 image: "hitbybusJan13th.jpg"
 title: "What Happens to Your Cryptocurrency If You Get Hit By a Bus?"
-original-author: t0dd  
+original-author: t0dd
 original-link: https://www.dashforcenews.com/happens-cryptocurrency-get-hit-bus/
 ---
 
 Or, how to manage volatility's ultimate endstate: Death.
 
 In a previous article, I shared some guidance on how to introduce new people to Dash and other cryptocurrencies: "[Introducing FOMO Friends and Family to Cryptocurrencies](https://www.dashforcenews.com/introducing-fomo-friends-family-cryptocurrencies/)". Within that article, we touched upon end-of-life matters. In this article, we expand that topic and give it a bit more consideration.
 
-Answer this question: What happens to your cryptocurrency hodlings if you get hit by a bus?
```

Comment:
I think this particular "typo" might be intentional. ;-)

Reply:
OK, so revert this one ?

Reply:
Well, let's wait to see what @tungfa says on it.

Reply:
hey guys
"holdings" sounds good to me ! (am not native english)